test_programs/plonky2_trace/2_function_calls/expected_trace.json

```diff
@@ -0,0 +1,22 @@
+[
+  { "Path": "" },
```

Comment:
Path to what? Should it be empty?

Reply:
This is an artifact of the library that generates the structure: https://github.com/metacraft-labs/runtime_tracing?tag=0.5.4

Reply:
Thanks